### PR TITLE
Pokemon Emerald: Change "settings" to "options" in docs

### PR DESCRIPTION
**Pull request by Zunawe**
## What is this fixing or adding?

Brings the Emerald docs in line with #2037.

## How was this tested?

Ran web host, checked pages, clicked links.

### CODE REVIEW

**Review by ReverM — CHANGES_REQUESTED** [review]
There seems to be two instance of the word setting left in the info doc
one is at the line 31 (with a setting it can..) and the other is at line 46 (depending on your receive item message setting)

**Review by ReverM — APPROVED** [review]
LGTM

File: worlds/pokemon_emerald/docs/en_Pokemon Emerald.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ randomizer. Here are some of the more important ones:
 - You can have both bikes simultaneously
 - You can run or bike (almost) anywhere
 - The Wally catching tutorial is skipped
-- All text is instant, and with a setting it can be automatically progressed by holding A
+- All text is instant, and with an option it can be automatically progressed by holding A
```
Comment (alwaysintreble) [inline worlds/pokemon_emerald/docs/en_Pokemon Emerald.md:31]:
```suggestion
- All text is instant and, with an option, can be automatically progressed by holding A

**Review by Exempt-Medic — APPROVED** [review]
LGTM

**Comment by ReverM** [comment]
I have covered the changes in https://github.com/ArchipelagoMW/Archipelago/pull/2622, maybe this PR could be closed then?

**Comment by Zunawe** [comment]
This already has reviews and doesn't need to wait on the additional approvals of all other world maintainers, so I think it makes more sense to keep this one.

**Comment by alwaysintreble** [comment]
This pr is also from the world maintainer so takes precedence imo

**Review by black-sliver — APPROVED** [review]

**Pull request merged**